### PR TITLE
Update tailscaled.in

### PR DESCRIPTION
**Pull request by yejunhuang**
Follow https://tailscale.com/kb/1097/install-opnsense/ .

/etc/rc.conf is set as follows:
tailscaled_enable="YES"
tailscaled_up_args="--accept-dns=false --advertise-routes=192.168.2.0/24"

After reboot, tailscale on the external network cannot connect to the LAN. At this point everything works fine after running the following line: /usr/local/bin/tailscale up --accept-dns=false --advertise-routes=192.168.2.0/24

After many attempts, it was found that /usr/local/etc/rc.d/tailscaled needs to be modified and depends on miniupnpd.

### CODE REVIEW

**Comment by fichtner** [comment]
Huh? miniupnpd doesn’t use the rc system with OPNsense and this generic type of change to a very specific dependency is quite strange.

**Comment by fichtner** [comment]
Also changes for ports should go to FreeBSD, which will discard this for the same reasons mentioned above.

**Comment by yejunhuang** [comment]
非网关的FreeBSD Host安装tailscale并不需要在本机安装miniupnpd，只要2个对端不都是在网关不支持NAT-PMP的局域网内，就能正常通信。
即使在OPNsense上安装tailscale，也不一定需要安装miniupnpd，只要对端在互联网上或在支持NAT-PMP的局域网内，就能正常通信。
但或许多数情况下，如果在OPNsense上安装了tailscale，那么也会安装miniupnpd，会希望miniupnpd先于tailscale启动，tailscale利用miniupnpd提供的NAT-PMP功能。
综上所述，在OPNsense上安装tailscale，与在FreeBSD上安装tailscale有点不同，是一种特殊情况，特殊到可以单独做一个不同的tailscale Port。
这个特殊的tailscale Port将考虑到与miniupnpd协同，后于miniupnpd启动。


Installing tailscale on a non-gateway FreeBSD Host does not require miniupnpd to be installed on the local machine. As long as the two peers are not both in a LAN where the gateway does not support NAT-PMP, they can communicate normally.

Even if you install tailscale on OPNsense, you do not necessarily need to install miniupnpd. As long as the peer is on the Internet or in a LAN that supports NAT-PMP, you can communicate normally.

But perhaps in most cases, if tailscale is installed on OPNsense, miniupnpd will also be installed. You will want miniupnpd to start before tailscale, and tailscale uses the NAT-PMP function provided by miniupnpd.

To sum up, installing tailscale on OPNsense is a little different from installing tailscale on FreeBSD. It is a special situation, so special that you can create a different tailscale port separately.
This special tailscale port will be considered for cooperation with miniupnpd and will be started after miniupnpd.
